File: apps/api/routes/root.go
package routes

import (
	"fmt"

	"github.com/gofiber/fiber/v2"
)

func root(c *fiber.Ctx) error {
	fmt.Println(c.Get("Authorization"))

	return c.JSON(fiber.Map{
		"status":  "ok",
		"message": "hello world!",
	})
}

func health(c *fiber.Ctx) error {
	fmt.Println(c.Get("Authorization"))
	return c.JSON(fiber.Map{
		"status":  "ok",
		"message": "Server is running",
	})
}

File: server/src/routes/root.ts
import { Hono } from 'hono'

const router = new Hono()

router.get('/', (c) => {
    return c.json({ status: 'ok' })
})

router.get('/health', (c) => {
    return c.json({ status: 'ok' })
})

router.get('/api/health', (c) => {
    return c.json({ status: 'ok' })
})

export const rootRouter = router
